new updates for the fixed footer

SamLaiiii opened the pull request:
Fixes #  112

## Description of the Change
Add the style of the body in the base.html file
## Checklist

Please check all that apply to your proposed changes

<!-- Replace '[ ]' with '[x]' to indicate that the checklist item is completed. -->
<!-- You can check the boxes now or later by just clicking on them. -->

- [x] HTML code change
- [ ] Added package dependency
- [ ] Added data
- [ ] Changed django model
- [ ] Documentation change
- [ ] Added or changed TaskRunner

### **Additional context**
<!-- Add any other context or additional information about the problem here.-->

SamLaiiii committed:
new updates for the fixed footer

SamLaiiii committed:
new updates for the fixed footer

SamLaiiii committed:
new updates for the fixed footer, including the blocking part

SamLaiiii committed:
new updates for the fixed footer, including the blocking part at the buttom

SamLaiiii committed:
Merge branch 'main' into Lai

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

astrophpeter reviewed (commented):
Nice work Sam, could you explain a bit about the problem that this fixes on the pull request? Just so we have a record :)

SamLaiiii commented:
The original problem is that the footer does not stick at the bottom of the browser. So, here is my solution.
I followed the path: app/host/templates/host/base.html. At first, I added the html style for both body and footer. After that, the specific style for body is this one:
     
       <style>
            body{
            padding: 20px;
            margin-bottom: 20px;
            border-radius: 5px;
                            }            
        </style>
For the footer:

            <style>
                .footer {
                position: FIXED;
                padding: 20px;
                left: 0;
                bottom: 0;
                width: 100%;
                background-color: white;
                color: white;
                text-align: center;
                            }
            </style>
In further, if someone wanted to changed the colors of the background, edit the background-color

astrophpeter reviewed (approved):
Awesome, thanks for fixing that sam :)!
